web-client/src/js/view/homePage/ResultPanel.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Row, Col, div} from 'react-bootstrap';
+import GridHeader from '../../gridHeader/GridHeader'
+
+class ResultPanel extends Component {
```

Comment:
Rename the class to Deployment Configuration. This panel is not meant to view Results.